bats/fb-test-puppet.bats

```diff
@@ -6,6 +6,10 @@ set -o pipefail
 load os_helper
 load foreman_helper
 
+if [[ -e /etc/profile.d/puppet-agent.sh ]] ; then
+  . /etc/profile.d/puppet-agent.sh
```

Comment:
Where does this file come from?

Reply:
The puppet package installs this. Since all the binaries are in /opt/puppetlabs/bin this ensures ```$PATH``` is set up correctly. It's a workaround since non-interactive shells don't source ```/etc/profile.d/*.sh``` and I wasn't sure how to do this otherwise when running via ansible.